content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -558,7 +558,7 @@ If the access modes are specified as ReadWriteOncePod, the volume is constrained
 | AzureFile            | &#x2713;               | &#x2713;              | &#x2713;      | -                      |
 | AzureDisk            | &#x2713;               | -                     | -             | -                      |
 | CephFS               | &#x2713;               | &#x2713;              | &#x2713;      | -                      |
-| Cinder               | &#x2713;               | -                     | -             | -                      |
+| Cinder               | &#x2713;               | -                     | ([if multi-attach volumes are available](https://github.com/kubernetes/cloud-provider-openstack/issues/1315))            | -                      |
```

Comment:
On this issue, there is the following link to the multi-attach doc, but it is on someone's private repo?  Is it somewhere in a merged official doc?  Also in this doc, it uses CSC tools from gocsi which is no longer supported. Do you have plan to use a different tool?

https://github.com/ramineni/cloud-provider-openstack/blob/953450377f722274295c7a27536c9afa772f1bc7/docs/using-cinder-csi-plugin.md#multi-attach-volumes

Reply:
The official docs do mention multi-attached cinder volumes.

https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/features.md#multi-attach-volumes

The official documentation is lacking on how one creates these. https://docs.openstack.org/cinder/latest/admin/volume-multiattach.html does go into more details with the `cinder` cli.

I hadn't noticed that the link in that GH issue was on a fork of the docs. I'll switch to the official docs.

Thanks Xing.